frontend/public/components/utils/name-value-editor.jsx

```diff
@@ -63,14 +63,14 @@ export const NameValueEditor = DragDropContext(HTML5Backend)(class NameValueEdit
       const key = _.get(pair, [NameValueEditorPair.Index], i);
       return <PairElement onChange={this._change} index={i} nameString={nameString} valueString={valueString} allowSorting={allowSorting} readOnly={readOnly} pair={pair} key={key} onRemove={this._remove} onMove={this._move} rowSourceId={nameValueId} configMaps={configMaps} secrets={secrets} />;
     });
-    return <div>
+    return <div className="container-fluid pairs-list__container">
```

Comment:
I'd change this to a <React.Fragment> since you don't actually need the container-fluid and pairs-list__container override.

Reply:
nice.